reflex/state.py

```diff
@@ -1031,6 +1245,10 @@ class StateUpdate(Base):
     final: bool = True
 
 
+class LockExpiredError(Exception):
```

Comment:
We started a new file `reflex/utils/exceptions.py`, should we move the error definitions there or keep them?